test/CustomerClass-test.js

```diff
@@ -0,0 +1,355 @@
+    bedSize: 'queen',
+    numBeds: 2,
+    costPerNight: 176.36 } ]
+)
+})
+it('should return avalible bookings by suite', () => {
+    expect(customer.availableToday(customer.bookings("2019/11/06"), 'suite')).to.deep.equal([ { number: 2,
+    roomType: 'suite',
+    bidet: false,
+    bedSize: 'full',
+    numBeds: 2,
+    costPerNight: 477.38 },
+  { number: 10,
+    roomType: 'suite',
+    bidet: false,
+    bedSize: 'twin',
+    numBeds: 1,
+    costPerNight: 497.64 },
+  { number: 24,
+    roomType: 'suite',
+    bidet: false,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 327.24 } ]
+)
+})
+
+it('should return avalible bookings by junior suite', () => {
+    expect(customer.availableToday(customer.bookings("2019/11/06"), 'junior suite')).to.deep.equal( [{ number: 6,
+    roomType: 'junior suite',
+    bidet: true,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 397.02 },
+  { number: 8,
+    roomType: 'junior suite',
+    bidet: false,
+    bedSize: 'king',
+    numBeds: 1,
+    costPerNight: 261.26 },
+  { number: 17,
+    roomType: 'junior suite',
+    bidet: false,
+    bedSize: 'twin',
+    numBeds: 2,
+    costPerNight: 328.15 } ])
+  })
+
+it('should return avalible bookings by single room', () => {
+    expect(customer.availableToday(customer.bookings("2019/11/06"), 'single room')).to.deep.equal([ { number: 3,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'king',
+    numBeds: 1,
+    costPerNight: 491.14 },
+  { number: 4,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 429.44 },
+  { number: 5,
+    roomType: 'single room',
+    bidet: true,
+    bedSize: 'queen',
+    numBeds: 2,
+    costPerNight: 340.17 },
+  { number: 9,
+    roomType: 'single room',
+    bidet: true,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 200.39 },
+  { number: 11,
+    roomType: 'single room',
+    bidet: true,
+    bedSize: 'twin',
+    numBeds: 2,
+    costPerNight: 207.24 },
+  { number: 12,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'twin',
+    numBeds: 2,
+    costPerNight: 172.09 },
+  { number: 16,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'full',
+    numBeds: 2,
+    costPerNight: 325.6 },
+  { number: 19,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 374.67 },
+  { number: 21,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'full',
+    numBeds: 2,
+    costPerNight: 429.32 },
+  { number: 22,
+    roomType: 'single room',
+    bidet: false,
+    bedSize: 'full',
+    numBeds: 2,
+    costPerNight: 350.31 },
+  { number: 25,
+    roomType: 'single room',
+    bidet: true,
+    bedSize: 'queen',
+    numBeds: 1,
+    costPerNight: 305.85 } ]
+)
+});
+
+
+  it('should return todays date', () => {
+    //there needs to be a better way to test this
+    expect(customer.todaysDate()).to.equal('2019/11/03');
+  });
+
```

Comment:
I'm not sure on other ways to test this. Mine is calculated in my index.js file so I don't have any testing for it.